rules.yaml

```diff
@@ -639,6 +639,18 @@ rules:
         key: TASKRC
         value: ${XDG_CONFIG_HOME}/task/taskrc
 
+  - name: texworks
+    dotfile:
+      name: .TeXworks
+      is_dir: true
+    actions:
+      - type: migrate
+        source: ${HOME}/.TeXworks
+        dest: ${XDG_CONFIG_HOME}/TeXworks
```

Comment:
I see in https://www.systutorials.com/docs/linux/man/1-texworks/ that the configuration file reside in `~/.config/TUG/TeXworks.conf` and `TW_LIBPATH` should point to the "additional configuration and library files". Maybe `${XDG_CONFIG_HOME}/TUG/conf.d` or something is more appropriate?

Reply:
That's a better location for sure, but it's not the first place you'd look. Perhaps `${XDG_DATA_HOME}/TeXworks` as this contains library files too?

Reply:
It does. But I am not sure what is better for users who backup configuration files. Not to backup the app's files or to back them up with some data files. I think the first option is better unless these lib files are too big.